### PR TITLE
Full Panasonic DMC-GM5 support

### PR DESCRIPTION
**Pull request by digint**
Added complete support for "Panasonic DMC-GM5":
- All aspect ratio modes (extracted from EXIF data)
- White balance presets (extracted from EXIF data)
- Noise profiles

Note that for "Extended ISO" mode (GM5 specific option allowing ISO < 200), the sensor produces non-linear data, which is not supported by rawspeed and result in purple-ish highlights.

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -2647,6 +2647,37 @@
           ]
         },
         {
+          "comment": "dmc-gm5 contributed by axel burri",
```
Comment (LebedevRI) [inline data/noiseprofiles.json]: We need to review those PDF graphs generated while creating this profile.
Please upload resulting dt-noiseprofile-XXXXXXXX.tar.gz

Comment (digint) [inline data/noiseprofiles.json]: The file can be downloaded from here:
http://dev.tty0.ch/dt-noiseprofile-20150425.tar.gz

**Comment by pmjdebruijn** [comment]
@rawspeed, generally speaking we'd prefer negative width/height notation (which just means pixels to crop from right/bottom). Would you mind adjusting that?

**Comment by digint** [comment]
@pmjdebruijn: As Panasonic reports different sensor size for different aspect ratios, I thought it was more comprehensible with the "positive" notation. You might have other considerations to take into account, so if you really wish I will change it to "negative" later tonight.

EXIF (4:3):
```
Sensor Width                    : 4816
Sensor Height                   : 3464
```

EXIF (3:2):
```
Sensor Width                    : 4816
Sensor Height                   : 3080
```

**Pull request merged**